README.md

```diff
@@ -172,3 +182,21 @@ the API documentation in [toxcore/tox.h](toxcore/tox.h) for more information.
 
 - [Another echo bot](https://wiki.tox.chat/developers/client_examples/echo_bot)
 - [minitox](https://github.com/hqwrong/minitox) (A minimal tox client)
+
+## SAST Tools
+
+- [clang-tidy](https://clang.llvm.org/extra/clang-tidy/): A clang-based C++
+  "linter" tool.
+- [Coverity](https://scan.coverity.com/): Find and fix defects in your Java,
+  C/C++, C#, JavaScript, Ruby, or Python open source project for free.
+- [cppcheck](https://cppcheck.sourceforge.io/): A static analysis tool for C/C++
```

Comment:
All others are called "a static analyzers" but this one is "a static analysis tool", kind of inconsistent.

Reply:
Also copied from their website. But sure, fixed.